[BE-3926]: Added new step add for review doc, and added app informati… (#516)
appcircleio/appcircle-docusaurus: docs/publish-module/publish-information/app-information.md

---
title: App Information
description: Learn how to update App Information in the Publish module of Appcircle
tags: [publish, publish module, app information, app store connect, review]
sidebar_position: 9
---

import Screenshot from '@site/src/components/Screenshot';

# App Information

> In order for a binary to be successfully sent to the review, some information must be completed. With Appcircle's App Information feature, you can update the information required for binary submission.

Reviewer: ```suggestion
For a binary to be successfully sent for review, certain information must be completed. By using Appcircle's App Information feature, you can update the required information for binary submission.
```

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3926-appInfodot.png' />

### Localizable Informations

The localization dropdown allows you to select the language in which you want to present your app’s information on the App Store. This feature supports multiple languages, ensuring that you can target specific demographics and cater to a global audience.

When you select a language, you will provide localized versions of your app's information, including name, subtitle, and privacy policies. Localization helps in reaching a wider audience by providing information in the users' native language.

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3926-localizeInfo.png' />

### General Informations

General information area allows you to see and update some of your information that will appear on the market for your application. 

> In these fields, you can determine the category of your application and fill in the content right information it contains.

Reviewer: ```suggestion
In these fields, you can specify the category of your application and provide information about the content it contains.
```

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3926-generalInfo.png' />

### Update and Save

> You can instantly see your current App Information details and if you wish, you can simultaneously update these values on your App Store Connect account. Each time this screen is opened, the current information will be retrieved.

Reviewer: ```suggestion
You can instantly view your current App Information details and, if desired, simultaneously update these values on your App Store Connect account. Each time this screen is opened, the current information will be retrieved.
```

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3926-appInfoDetails.png' />

:::danger App Information Save

> When you make a change to this informations, when you click the **save button**, the changes you have made will be updated on your App Store Connect account **at that moment**. Make sure your selections are **correct** and the information entered is **accurate**.

Reviewer: ```suggestion
When you make a change to these informations and click the **save button**, the updates will be **immediately** applied to your App Store Connect account. Ensure your selections are **correct** and the information entered is **accurate**.
```
